_includes/update-payment-order.md

```diff
@@ -1,5 +1,11 @@
 ### Update Payment Order
 
+{% include alert.html type="warning" icon="warning" body="If a payment is
+attempted, failed or currently being processed, you will no longer be able to
```

Comment:
Can you write a date to indicate from when "no longer" applies?

Reply:
Or perhaps just remove it, if this is a permanent change.

Reply:
We're hoping to fix this to allow changes to orderItems down the line, but it will take a while to implement the fix for every one of the supported UIs. To ensure no missmatches with orderItem and actually authorized amount, this will be a short term solution.

I'll remove the "no longer" clause. When the change goes out depends on the referenced payment order PR under "related PR" is merged, tested and deployed.
We will also send out an email to support to inform merchants of the change as well.